### DISCUSSION TITLE
Handle request logging

### DISCUSSION THREAD

**Discussion by ArchercodC** [dotnet/yarp#2135] [Q&A] [answered]
Is it possible to prevent logging all parts of request, instead leaving only request finishing log that has info about execution time if I am using YARP Direct Proxy Example? If it`s possible will be appreciated to see solution

**Comment by samsp-msft** [comment, marked as answer]
If you are doing direct forwarding, rather than using the pipeline, you have control over the timing of the beginning and end of the request, so turn of YARP logging (set its level to None) and log the message that you want?
